### DISCUSSION TITLE
NPCBots not entering in Raids or Battlegrounds

### DISCUSSION THREAD

**Discussion by acidicp** [trickerer/Trinity-Bots#834] [Q&A] [answered]
I create all of my bots, get them geared up, got in a party and raid, raid frame shows all 10 slots filled. When I go to enter a raid, for instance Naxxramas, all of the bots are with me at the portal, set to follow, and when I enter the raid the bots do not appear inside. Their portraits in my party frame are blank and they are still in the party and the raid. Is this a known issue with a known fix, or does anyone have any ideas how I could troubleshoot the issue?  I have looked around and have not found anyone else with the issue or a fix for it.
Edit: Meant to mention this also occurs when I join a battleground as a party. I'm less concerned about this but just in case it helps identify a solution I wanted to include it.

**Comment by trickerer** [comment, marked as answer]
Check your worldserver.conf and see if raid maps and others are enabled: `NpcBot.Enable.Raid`, `NpcBot.Enable.BG`, etc.

**Reply by acidicp** [reply]
This worked! thanks for being so responsive, and sorry it took me a while to get back!
